README.md

```diff
@@ -79,6 +81,34 @@ then add this to your maven configuration:
 </dependencies>
 ```
 
+### Gradle and Android Studio
+
+By default, Android Studio will not be able to find and index the sources
+generated by the `dagger-compiler` if it is a `provided` dependency.
+Various `apt` (think Annotation Processing Tool, not Advanced Packaging Tool)
```

Comment:
This isn't quite accurate to say since Annotation Processing Tool is not what the dagger-compiler uses. It's a defunct pre-JSR 269 standalone tool.